src/routes/home/Home.js

```diff
@@ -149,11 +149,22 @@ async function extractDate({ attachmentFile, attachmentArrayBuffer, ext }) {
       throw new Error(`${attachmentFile.name} is not an image/video`);
     }
 
-    const { CreateDate } = await exifr.parse(attachmentArrayBuffer, [
+    const {
+      CreateDate,
+      OffsetTimeDigitized,
+    } = await exifr.parse(attachmentArrayBuffer, [
       'CreateDate',
+      'OffsetTimeDigitized',
     ]);
 
-    return CreateDate.getTime();
+    // console.log({ CreateDate, OffsetTimeDigitized });
+
+    return {
+      millisecondsSinceEpoch: CreateDate.getTime(),
+      offset: OffsetTimeDigitized
+        ? parseInt(OffsetTimeDigitized, 10) * -60
+        : new Date().getTimezoneOffset(),
```

Comment:
may want to dedupe this with the other instance of `new Date().getTimezoneOffset()`